packages/mobile/src/App.tsx

```diff
@@ -58,6 +58,19 @@ const App = () => {
     setIsReadyToSetupBackend(!entropy)
   }, [])
 
+  useEffectOnce(() => {
+    refreshConnectivity()
+  })
+
+  useAppState(
```

Comment:
Could we update this hook to not require both, if we only need to add callback for one of them?

Reply:
Also considering we have this here too, i wonder if this could also be moved to a saga. Really not a big deal though, maybe this is a good topic for our sync this week

Reply:
Updated the hook. Let me tinker with the saga approach. I need to learn more about them anyway :)